service/controllers/messages.go

```diff
@@ -115,7 +115,19 @@ func routes(routeObject response, w http.ResponseWriter) {
 
 	} else if strings.ToLower(firstPars) == "bing" {
 		query := "https://www.bing.com/search?q=" + messageExceptFirstPars
-		HandlerBing("GET", query)
+		result := HandlerBing("GET", query)
+
+		// processing
+
+		response := processBingResponses(result)
+		responseJSON := jsonResponseQuery{
+			Status: true,
+			Message: "query-result",
+			Result: response,
+		}
+		jData, _ := json.Marshal(responseJSON)
+		w.Write(jData)
+
```

Comment:
make a call to the Text to Speech in controllers.go, **after you send back the response**.